rust/repositories.bzl

```diff
@@ -33,6 +33,16 @@ DEFAULT_TOOLCHAIN_TRIPLES = {
 
 def rules_rust_dependencies():
     """Dependencies used in the implementation of `rules_rust`."""
+
+    maybe(
+        http_archive,
+        name = "platforms",
+        urls = [
+            "https://mirror.bazel.build/github.com/bazelbuild/platforms/releases/download/0.0.5/platforms-0.0.5.tar.gz",
+            "https://github.com/bazelbuild/platforms/releases/download/0.0.5/platforms-0.0.5.tar.gz",
+        ],
+        sha256 = "379113459b0feaf6bfbb584a91874c065078aa673222846ac765f86661c27407",
+    )
```

Comment:
I could not determine what was responsible for loading `bazelbuild/platforms` and I need 0.0.5 or newer because it has the `cpu` constraint.

Is there a better place to do this?

Reply:
Hmm, have you updated this value in Bazel itself? I think I’d defer to @hlopko on whether or not we should add an explicit dependency on the platforms repository. But this seems like the right spot

Reply:
Its not clear to me that bazelbuild/platforms is a built-in in bazel itself - I couldn't find where it gets loaded.

Reply:
Ah, the thing I'm referring to is already done: https://github.com/bazelbuild/bazel/commit/676a0c8dea0e7782e47a386396e386a51566087f

I'm fine with this PR then, sorry for the delay.